angelsaddons-cab/control.lua

```diff
@@ -82,7 +80,8 @@ script.on_event(defines.events.on_built_entity, function(event)
      event.created_entity.name == global.vehicleData.entityName then
   end
 end)
-]]--
+]]
+--
```

Comment:
It is currently breaking multiline comments

Reply:
A comment starts anywhere with a double hyphen (--) and runs until the end of the line. Lua also offers block comments, which start with --[[ and run until the corresponding ]]

I guess thats why the formatter is making a line break. The two -- in line 84 are not needed and actually starting a line comment.

Reply:
```suggestion
]]
```